files/tarball_ingester_userdata.tpl

```diff
@@ -80,15 +81,47 @@ aws s3 cp s3://${s3_artefact_bucket}/dataworks-tarball-ingester/dataworks-tarbal
     /tmp/dataworks-tarball-ingester-${tarball_ingester_release}.zip
    unzip -d /opt/tarball_ingestion /tmp/dataworks-tarball-ingester-${tarball_ingester_release}.zip
    chmod u+x /opt/tarball_ingestion/file-transfer.sh
+   chmod u+x /opt/tarball_ingestion/steps/copy_collections_to_s3.py
 
 echo "Changing permissions and moving files for tarball ingester"
 chown tarball_ingestion:tarball_ingestion -R  /opt/tarball_ingestion
 chown tarball_ingestion:tarball_ingestion -R  /var/log/tarball_ingestion
 
+echo "Installing Python3 for running encryption script"
+yum install -y python3
+
 if [[ "${environment_name}" != "production" ]]; then
     echo "Running script to copy synthetic tarballs..."
     echo "Synthetic tarball script would have run" >> /var/log/tarball_ingestion/tarball_ingestion.out 2>&1
 fi
 
-echo "Execute Python script to process data..."
-echo "Process data script would have run" >> /var/log/tarball_ingestion/tarball_ingestion.out 2>&1
+TI_ASG_NAME=$(aws autoscaling describe-auto-scaling-instances \
+    --instance-ids $INSTANCE_ID \
+    --region $AWS_DEFAULT_REGION \
+    | grep AutoScalingGroupName | cut -d'"' -f4)
+
+
+echo "Execute Python script to process Incrementals collections data..."
+python3 /opt/tarball_ingestion/steps/copy_collections_to_s3.py -s "${ti_src_dir}" \
+    -s3b "${ti_s3_bucket}" \
+    -s3p "${ti_s3_prefix}" \
+    -m "${ti_manifest_path}" \
+    -t "${ti_tmp_dir}" \
+    -d "${dks_endpoint}" \
+    -f "incrementals" \
+    -w "${ti_wait}" \
+    -i "${ti_interval}" \
+    -a "$TI_ASG_NAME" >> /var/log/tarball_ingestion/tarball_ingestion.out 2>&1
+
+
+echo "Execute Python script to process Full collections data..."
+python3 /opt/tarball_ingestion/steps/copy_collections_to_s3.py -s "${ti_src_dir}" \
+    -s3b "${ti_s3_bucket}" \
+    -s3p "${ti_s3_prefix}" \
+    -m "${ti_manifest_path}" \
+    -t "${ti_tmp_dir}" \
+    -d "${dks_endpoint}" \
+    -f "full" \
```

Comment:
this should be fulls,  as discussed this should represnt the key in manifest file